scripts/emulator-testing/emulators/database-emulator.ts

```diff
@@ -41,7 +41,7 @@ export class DatabaseEmulator extends Emulator {
         {
           uri: `http://localhost:${this.port}/.settings/rules.json?ns=${this.namespace}`,
           headers: { Authorization: 'Bearer owner' },
-          body: '{ "rules": { ".read": true, ".write": true } }'
+          body: '{ "rules": { ".read": true, ".write": true, "testing": { ".indexOn": "testIndex" } } }'
```

Comment:
It's probably worth dropping a reference to the CI database.rules.json so that future authors know to keep them in sync.